CppImport/src/ClangHelpers.cpp

```diff
@@ -290,4 +293,57 @@ void ClangHelpers::attachDeclarationComments(clang::NamedDecl* namedDecl, Model:
 	}
 }
 
+void ClangHelpers::storeNodeFilename(Model::Node* node, clang::SourceLocation location)
+{
+	auto fullPath = presumedFilenameWithoutExtension(location);
+	nodeToFilenameMap_ [node] = fullPath.right(fullPath.size() - rootProjectPath_.size() - 1);
+}
+
+void ClangHelpers::storeNodeFilename(Model::Node* node, Model::Node* from)
+{
+	auto it = nodeToFilenameMap_.find(from);
+	Q_ASSERT(it != nodeToFilenameMap_.end());
+	nodeToFilenameMap_[node] = *it;
+}
+
+void ClangHelpers::dumpMergeMap(QString filename)
+{
+	QHash<QString, QString> dummy;
```

Comment:
This `dummy` even if unused has a property. Why not call it, `automaticallyGeneratedMap` ?